src/test/fixtures/profiles/gecko-profile.js

```diff
@@ -581,8 +581,9 @@ export function createGeckoCounters(thread: GeckoThread): GeckoCounter {
     // Go through all the thread samples and create a corresponding counter entry.
     const time = thread.samples.data[i][1];
     // Create some arbitrary data.
-    const number = Math.sin(i / 50);
-    const count = Math.sin(i / 100);
+    const number = Math.sin(i);
+    // Counts are integers, make them range as int values 0 to 100.
```

Comment:
I believe `number` should be integer and positive (and it isn't here), while `count` isn't necessarily. (like you did in `getCounter` below)